playstore.coffee

```diff
@@ -0,0 +1,90 @@
+# Description:
+#   All things Google's Playstore
+#
+#   Set your Google Playstore application id as an
+#   environment variable `PLAYSTORE_APP_ID`
+#   e.g. PLAYSTORE_APP_ID=com.nextdoor
+#
+# Dependencies:
+#   cheerio
```

Comment:
Good question - not sure how to best approach this. One part of me just wants to open source the whole repo. Perhaps, for now, let's publish one by one